### PR TITLE
Fix #3154 - Fixes the UI inconsistency in README.md file

### LINKED ISSUE #3154: UI inconsistency in README.md file
## Description
All steps under "Building from source" section in the README.md file have the same layout except the last and first step.

## Steps to Reproduce

open the README.md file in this repository.
## Expected Behaviour

All the steps should have the same layout in the Building from source section
## Actual Behaviour

Some steps do not follow the same UI pattern.
## Settings

- Mifos X version: 
- Browser used:
- OS:

### PR DESCRIPTION
**Pull request by OHassini**
## Description
Fixes the layout of the first and the last step in the "Building from source" section in the README.md
file.
## Related issues and discussion
#3154 

## Screenshots, if any
A summary of the changes :
Capture d’écran (167)

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Validate the JS and HTML files with `grunt validate` to detect errors and potential problems in JavaScript code.

- [x] Run the tests by opening `test/SpecRunner.html` in the browser to make sure you didn't break anything.

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `community-app/Contributing.md`.

### CODE REVIEW

**Comment by OHassini** [comment]
@luckyman20 Hey : ), Please have a look at this.

File: README.md [hunk 1]
```diff
@@ -54,8 +58,14 @@ This is the default web application built on top of the MifosX platform for the
     Waiting...Warning: EMFILE, too many open files
 
    ```
+   
+1. You can use these credentials to log in
 
-1. Default username/password: mifos/password. This application will hit the demo server by default.
+   ```
+    Username: fineractCn
```
Comment (luckyman20) [inline README.md]:
```suggestion
    Username: mifos
```

**Review by luckyman20 — CHANGES_REQUESTED** [review]

**Comment by OHassini** [comment]
Oh my god, how fool of me, I am really sorry I was working on a mifos-mobile-cn task right before making this commit, I just updated the README file again with the changes and squashed my commits. @luckyman20

File: README.md [hunk 2]
```diff
@@ -54,8 +58,14 @@ This is the default web application built on top of the MifosX platform for the
...
    ```
+   
+1. You can use these credentials to log in
 
-1. Default username/password: mifos/password. This application will hit the demo server by default.
+   ```
+    Username: mifos
+    Password: password
+   ```
+This application will hit the demo server by default.
```
Comment (luckyman20) [inline README.md]: Keep this line at the end as a note.

Comment (OussEmaDevCode) [reply README.md]: the end of the README file or the "Building from source" section ?

**Comment by OHassini** [comment]
@luckyman20 done!!

**Review by luckyman20 — APPROVED** [review]
Looks good!

**Pull request merged**